src/styles/blocks/card.scss

```
@@ -0,0 +1,123 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  position: relative;
+  border-radius: 5px;
+  border: 1px solid #F3F3F3;
+  background: $White;
+
+  &__product {
+    display: block;
+    height: 206px;
+
+    &-img {
+      padding: 32px 19px 40px;
+      width: 160px;
+      height: 134px;
+      flex-shrink: 0;
+    }
+  }
+
+  &__name {
+    margin: 0;
+    padding: 0 16px;
+    color: $Main-accent;
+    font-size: 12px;
+    font-style: normal;
+    font-weight: 500;
+    line-height: 150%;
+  }
+
+  &__code {
+    color: $Secondary;
+    padding: 0 16px;
+    margin-top: 4px;
+    font-size: 10px;
+    font-style: normal;
+    font-weight: 400;
+    line-height: 140%;
+  }
+
+  &__reviews {
+    margin-top: 16px;
+    padding: 0 16px;
+    height: 16px;
+    flex-shrink: 0;
+    display: flex;
+    justify-content: space-between;
+
+    &-review {
+      display: flex;
+      width: 53px;
+      height: 16px;
+      flex-direction: column;
+      justify-content: flex-end;
+      flex-shrink: 0;
+      color: $Main-accent;
+      text-align: right;
+      font-size: 10px;
+      font-style: normal;
+      font-weight: 400;
+      line-height: 14px;
+    }
+  }
+
+  &__price {
+    margin-top: 24px;
+    padding: 0 16px;
+    display: flex;
+    width: 166px;
+    height: 18px;
+    justify-content: space-between;
+    align-items: flex-end;
+
+    &-text {
+      color: $Secondary;
+      font-size: 12px;
+      font-style: normal;
+      font-weight: 400;
+      line-height: 18px; /* 150% */
```

```suggestion
      line-height: 18px; 
```
this is unnecessary comment

---

done, but I fixed first item through padding